oracle/README.md

```diff
@@ -403,69 +358,21 @@ SQL> select blocking_session,username,osuser, sid, serial#, wait_class, seconds_
 where blocking_session is not NULL order by blocking_session;
 ```
 
-3. Once configured, you can create a [monitor][12] based on `oracle.custom_query.locks` metrics.
+3. Once configured, you can create a [monitor][10] based on `oracle.custom_query.locks` metrics.
 
 ## Data Collected
 
 ### Metrics
 
-See [metadata.csv][13] for a list of metrics provided by this integration.
+See [metadata.csv][11] for a list of metrics provided by this integration.
 
 ### Events
 
 The Oracle Database check does not include any events.
 
 ### Service Checks
 
-See [service_checks.json][14] for a list of service checks provided by this integration.
-
-## Troubleshooting
-
-### Common problems
-#### Oracle Instant Client
-- Verify that both the Oracle Instant Client and SDK files are located in the same directory.
-The structure of the directory should look similar:
-
-```text
-|____sdk/
-|____network/
-|____libociei.dylib
-|____libocci.dylib
-|____libocci.dylib.10.1
-|____adrci
-|____uidrvci
-|____libclntsh.dylib.19.1
-|____ojdbc8.jar
-|____BASIC_README
-|____liboramysql19.dylib
-|____libocijdbc19.dylib
-|____libocci.dylib.19.1
-|____libclntsh.dylib
-|____xstreams.jar
-|____libclntsh.dylib.10.1
-|____libnnz19.dylib
-|____libclntshcore.dylib.19.1
-|____libocci.dylib.12.1
-|____libocci.dylib.18.1
-|____libclntsh.dylib.11.1
-|____BASIC_LICENSE
-|____SDK_LICENSE
-|____libocci.dylib.11.1
-|____libclntsh.dylib.12.1
-|____libclntsh.dylib.18.1
-|____ucp.jar
-|____genezi
-|____SDK_README
-
-```
-
-##### Linux
-- See further Linux installation documentation on [Oracle][2].
-
-##### Windows
-- Verify the Microsoft Visual Studio `<YEAR>` Redistributable requirement is met for your version. See the [Windows downloads page][15] for more details.
-- See further Windows installation documentation on [Oracle][4].
-
+See [service_checks.json][12] for a list of service checks provided by this integration.
```

Comment:
Could you add the `## Troubleshooting` header back in? This is required in order for the service checks to be properly rendered, and also adds context to the `#### JDBC driver (Linux only)` section. Screenshot of my local docs build for reference:
https://a.cl.ly/DOukNO9l